CenterForOpenScience/osf.io
# [#PLAT-710]Add node logs when users turn on/off request access

## Purpose
Add logs for enabling or disabling node access request


## Changes



## QA Notes
Create a projrect
go to the project settings page
select the tab access requests and check the box to "Allow users to request access to this project."
go back to the project overview page and check the log
go to the project settings page again
select the tab access requests and uncheck the box to "Allow users to request access to this project."
go back to the project overview page and check the log
Log expected to see:

![image](https://user-images.githubusercontent.com/4974056/39061688-226e8226-4493-11e8-9fb9-0e7ded252bc4.png)


## Documentation



## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-710
